dv/management/commands/import_news.py

```diff
@@ -42,12 +41,24 @@ def _save(self, item):
             news.summary = item['summary']
             news.image = item['image'].replace('http://', 'https://')
             news.is_partnership = item['is_partnership'] == 'yes'
-            if item['project_id']:
-                news.project_id = item['project_id'].strip()
+            project_id = item.get("project_id")
+            if project_id:
+                project = Project.objects.filter(code=project_id).exists()
+                if project:
```

Comment:
No need for the extra variable
`if Project.objects.filter(code=project_id).exists():`